WhaleSpotting/Services/SightingsService.cs

```diff
@@ -46,13 +46,23 @@ public async Task<List<SightingResponseModel>> GetAllSightings()
 
         public async Task<List<SightingResponseModel>> SearchSighting(SearchSightingRequestModel searchSighting, PageFilter pageFilter)
         {
+            var radiusInCoords = searchSighting.RadiusKm * 0.009;
+            var upperLatitude = searchSighting.Latitude + radiusInCoords;
+            var lowerLatitude = searchSighting.Latitude - radiusInCoords;
+
+            var upperLongitude = searchSighting.Longitude + radiusInCoords;
+            var lowerLongitude = searchSighting.Longitude - radiusInCoords;
+
             var sightings = await _context.Sightings
                 .Where(s => searchSighting.Species == null || s.Species == searchSighting.Species)
-                .Where(s => searchSighting.SightedFrom == null || s.SightedAt >= searchSighting.SightedFrom)
+                .Where(s => searchSighting.SightedFrom == null || s.SightedAt >= Convert.ToDateTime(searchSighting.SightedFrom))
                 .Where(s => searchSighting.SightedTo == null || s.SightedAt <= searchSighting.SightedTo)
```

Comment:
This should probably convert to date tiome as well?

Reply:
Not required for either